pglookout/cluster_monitor.py

```diff
@@ -44,7 +44,7 @@ class ReplicationSlot:
     state_data: str
 
 
-def wait_select(conn, timeout=5.0):
+def wait_select(conn, timeout=10.0):
```

Comment:
Let's bump this to 15s: https://aiven.atlassian.net/browse/PM-5174 required > 12s timeout. Eventually, this should be configurable or exponential backoff retrying instead of constantly staying put.